front/src/modules/ui/filter-n-sort/utils/getOperandLabel.ts

```diff
@@ -18,3 +18,21 @@ export function getOperandLabel(operand: FilterOperand | null | undefined) {
       return '';
   }
 }
+export function getOperandLabelShort(
+  operand: FilterOperand | null | undefined,
+) {
+  switch (operand) {
```

Comment:
I've discussed with @Bonapara , it's a good idea to also introduce '>' and '<'

However, in should have:
- is / contains: "Field: XXX"
- is-not / does-not-contain: "Field: Not XXX"
- greater-than / less-than: "Field > XXX" (without the dot)

Can we refactor the code so it also includes the ":" ?

```
case 'is':
    case 'contains':
      return ': ';
    case 'is-not':
    case 'does-not-contain':
      return ': Not ';
    case 'greater-than':
      return ' > ';
    case 'less-than':
      return ' < ';
    default:
      return ': ';

```